sfaira/commands/templates/multiple_datasets_not_streamlined/{{ cookiecutter.doi_sfaira_repr }}/{{ cookiecutter.id_without_doi }}.py

```diff
@@ -53,7 +53,7 @@ def __init__(
         # SFAIRA: name of column which contain streamlined cell ontology cell type classes:
         # self.obs_key_cellontology_original = x  # (optional)
 
-    def _load(self, fn):
-        if fn is None:
-            fn = os.path.join(self.path, self.directory_formatted_doi, "my.h5ad")
-        self.adata = ad.anndata.read(fn)
+    def _load(self):
+        # fn = os.path.join(self.data_dir, )  # SFAIRA ToDo: add the name of the raw file
+        # SFAIRA ToDo: add code that loads to raw file into an AnnData object
```

Comment:
@le-ander the `ToDo` statements must be fully capitalized. -> # `SFAIRA TODO:`

See: https://github.com/theislab/sfaira/pull/89/files#diff-aabee1516b72ed40ea1cc7ff2a9acd8325ad19c0bd55fffc4c589653fc8b7d21R111

Reply:
thanks for pointing that out, will change that.